plugins/gestures.koplugin/main.lua

```diff
@@ -120,6 +121,12 @@ Multiswipes allow you to perform complex gestures built up out of multiple swipe
 
 These advanced gestures consist of either straight swipes or diagonal swipes. To ensure accuracy, they can't be mixed.]])
 
+-- If the gesture contains the "toggle_touch_input" action, mark it always_active
+-- to make sure that InputContainer won't block it after the IgnoreTouchInput Event.
+function Gestures:isGestureAlwaysActive(ges)
+    return self.gestures[ges] and self.gestures[ges].toggle_touch_input or nil
```

Comment:
Note to self: the `or nil` is now redundant (I added it to clear the field instead of setting it to false; this is no longer a field ;)).